.circleci/config.yml

```diff
@@ -98,17 +104,11 @@ build_image: &build_image
   steps:
     - checkout
     - <<: *exports
-    - run: make VERSION="$VERSION" $IMAGE
+    - run: make VERSION="$DEV_TAG" $IMAGE
     - run: |
-        if [[ -z "$CIRCLE_PULL_REQUEST" ]]; then
+        if [[ -z "$CIRCLE_PULL_REQUEST" && -n "$DOCKER_USERNAME" && -n "$DOCKER_PASSWORD" ]]; then
```

Comment:
Why of the removal of the variable expansion {}?

BTW, a bash style guide that I try without much success to follow is this one https://google.github.io/styleguide/shell.xml#Variable_expansion